draft-ietf-quic-transport.md

```diff
@@ -6520,8 +6531,8 @@ of the security properties of QUIC depend on the TLS handshake providing these
 properties. Any attack on the TLS handshake could affect QUIC.
 
 Any attack on the TLS handshake that compromises the secrecy or uniqueness
-of session keys affects other security guarantees provided by QUIC that depends
-on these keys. For instance, migration ({{migration}}) depends on the efficacy
+of session keys, or the authentication of the participating peers, affects other security guarantees provided by QUIC that depend
+on those keys. For instance, migration ({{migration}}) depends on the efficacy
```

Comment:
I'm open to better phrasing suggestions here, too.

Reply:
The structure doesn't seem ideal, but this is OK with me and I can't see anything better.